examples/amp-story/animations-presets.html

```diff
@@ -0,0 +1,142 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    body {
+      font-family: sans-serif;
+    }
+    h1 {
+      font-family: Menlo, Monaco, monospace;
+      font-weight: bold;
+      font-size: 24px;
+      width: 100%;
+      text-align: left;
+    }
+    amp-story-page {
+      background: #ddd;
+    }
+    amp-story-grid-layer {
+      justify-items: center;
+    }
+    .square,
+    .circle {
+      background: #4285f4;
+      width: 80px;
+      height: 80px;
+    }
+    .circle {
+      border-radius: 80px;
+    }
+    </style>
+</head>
+<body>
+  <amp-story standalone>
+    <amp-story-page id="fade-in-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fade-in</h1>
+        <div class="circle"
+            animate-in="fade-in"
+            animate-in-duration="0.8s">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="twirl-in-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>twirl-in</h1>
+        <div class="square"
+            animate-in="twirl-in">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="fly-in-left-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fly-in-left</h1>
+        <div class="square"
+            animate-in="fly-in-left">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="fly-in-right-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fly-in-right</h1>
+        <div class="square"
+            animate-in="fly-in-right">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="fly-in-top-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fly-in-top</h1>
+        <div class="square"
+            animate-in="fly-in-top">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="fly-in-bottom-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fly-in-bottom</h1>
+        <div class="square"
+            animate-in="fly-in-bottom">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="rotate-in-left-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>rotate-in-left</h1>
+        <div class="square"
+            animate-in="rotate-in-left">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="rotate-in-right-page">
+      <amp-story-grid-layer template="vertical">
```

Comment:
this fails with an exception:

<img width="790" alt="screen shot 2017-10-10 at 2 16 00 pm" src="https://user-images.githubusercontent.com/380472/31388443-f5bbef0c-adc5-11e7-975a-2834af9f7bf7.png">

Reply:
Fixed by #11581